cl/donate/templates/donate.html

```diff
@@ -21,13 +21,15 @@
     let donor_info = $("#donor-info");
     let check_info = $("#check-info");
     let cc_info = $("#cc-info");
+    let bitcoin_info = $("#bitcoin-info");
```

Comment:
These changes bring back bitcoin. They shouldn't be here.

Reply:
this is a bad rebase I think.  I got mixed up about removing bitcoin no taking it away.